src/test/e2e/99_yolo_test.go

```diff
@@ -53,9 +53,17 @@ func TestDevDeploy(t *testing.T) {
 		return
 	}
 
+	// Generic test of dev deploy
 	stdOut, stdErr, err := e2e.Zarf(t, "dev", "deploy", "examples/dos-games")
 	require.NoError(t, err, stdOut, stdErr)
 
-	stdOut, stdErr, err = e2e.Zarf(t, "package", "remove", "dos-games", "--confirm")
+	stdOut, stdErr, err = e2e.Zarf(t, "tools", "kubectl", "delete", "namespace", "dos-games")
+	require.NoError(t, err, stdOut, stdErr)
```

Comment:
I might just be crazy but I don't think `package remove` works on something that was `dev deploy`ed, so I switched this to just delete the namespace instead...